src/client/components/home.js

```diff
@@ -178,98 +156,242 @@ class RequestForm extends Component {
   }
 }
 
+// N.b. scroller lists any doc in debug mode
+class Scroller extends Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      pagerLeftAvailable: false,
+      pagerRightAvailable: false,
+      docs: []
+    };
+
+    this.onScrollExplore = _.debounce(() => {
+      this.updatePagerAvailability();
+    }, 40);
+  }
+
+  componentDidMount(){
+    this.refreshDocs().then(() => this.onScrollExplore());
+  }
+
+  scrollExplore(factor = 1){
+    if( this.exploreDocsContainer ){
+      const container = this.exploreDocsContainer;
+      const padding = parseInt(getComputedStyle(container)['padding-left']);
+      const width = container.clientWidth - 2*padding;
+
+      this.exploreDocsContainer.scrollBy({
+        left: width * factor,
+        behavior: 'smooth'
+      });
+    }
+  }
+
+  scrollExploreLeft(){
+    this.scrollExplore(-1);
+  }
+
+  scrollExploreRight(){
+    this.scrollExplore(1);
+  }
+
+  updatePagerAvailability(){
+    if( this.exploreDocsContainer ){
+      const haveNoDocs = this.state.docs.length === 0;
+      const { scrollLeft, scrollWidth, clientWidth } = this.exploreDocsContainer;
+      const allTheWayLeft = scrollLeft === 0;
+      const allTheWayRight = scrollLeft + clientWidth >= scrollWidth;
+      let leftAvail = !allTheWayLeft && !haveNoDocs;
+      let rightAvail = !allTheWayRight && !haveNoDocs;
+
+      this.setState({
+        pagerLeftAvailable: leftAvail,
+        pagerRightAvailable: rightAvail
+      });
+    }
+  }
+
+  refreshDocs(){
+    const url = `/api/document`;
+
+    const toJson = res => res.json();
+    const update = docs => new Promise(resolve => this.setState({ docs }, () => resolve(docs)));
+    const doFetch = () => fetch(url);
+
+    return tryPromise(doFetch).then(toJson).then(update);
+  }
+
+  render(){
+    const exploreDocEntry = doc => {
+      const Article = doc.article.MedlineCitation.Article;
+      const title = Article.ArticleTitle;
+      let authorNames = Article.AuthorList.map(a => `${a.ForeName} ${a.LastName}`);
```

Comment:
To handle [case](https://www.ncbi.nlm.nih.gov/pubmed/30158200) where author is an organization: 
```js
let authorNames = Article.AuthorList.map(a => a.LastName ? `${a.ForeName} ${a.LastName}` : a.CollectiveName );
```